### DISCUSSION TITLE
keyboard shortcut for "Publish" Button. Improved workflow in the editor.

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22415] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be great if you added a new shortcut for the **publish button** in the editor. Currently there is only option to save using ctrl + s, but no option to publish.
I usually compare other page builders to Elementor. In one of the page builders, the keyboard shortcut ctrl + s publishes the changes. This usually speeds up the workflow in the editor. 

### Describe the solution you'd like

Please assign a simple keyboard shortcut (preferably ctrl + s) to the publish button. This will be a very important item in the editor. Using this keyboard shortcut, we can quickly publish editor changes.
Hi @rami-elementor, Since you are doing some keyboard shortcut for 3.14, maybe this is also useful. Please keep that in mind🙏

**Comment by milkbottlemotel** [comment]
I would also like to have such a keyboard shortcut!

**Comment by SkyeSoaring** [comment]
Need this, very annoying for development sites. If you're worried about accidentally publishing on a live site, make it an enabled option
